web/src/common.ts

```diff
@@ -16,6 +16,7 @@ const keys_map = new Map([
     ["Enter", "Return"],
     ["Ctrl", "⌘"],
     ["Alt", "⌥"],
+    ["Ctrl-[", "Cmd-["],
```

Comment:
We don't document keyboard shortcuts like this. We document each key separately.